BUILD-AND-DEPLOY.md

```diff
@@ -0,0 +1,28 @@
+# Build and Deploy
+
+## Front End 
+
+### Build configuration
+
+| Name                          | Description                                                                    | Required |
+| ----------------------------- | ------------------------------------------------------------------------------ | -------- |
+| `BABEL_ENV`                   | Sets the environment the code is running in                                    | Yes      |
+| `NODE_ENV`                    | Sets the environment the code is running in                                    | Yes      |
+| `NODE_PATH`                   | Folder path for the code folder structure                                      | Yes      |
+| `CI`                          | Switch that if is set to true will treat warnings as errors                    | No       |
+| `REACT_APP_EQ_AUTHOR_VERSION` | The current Author version. This is what gets reported on the /status endpoint | No       |
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.<br>
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.<br>
+Your app is ready to be deployed!
+
+See the section about [deployment](#deployment) for more information.
+
+### `yarn deploy`
+
+Builds (via `yarn build`) and deploys the project to Github Pages.
```

Comment:
This builds both the main app _and_ the Storybook app but only deploys the latter to GH Pages.